README.md

```diff
@@ -15,44 +15,52 @@ You can learn more on the [Okta + JavaScript][lang-landing] page in our document
 
 <!-- TOC depthFrom:2 depthTo:3 -->
 
-- [Getting started](#getting-started)
```

Comment:
It looks like there's an auto-TOC generator that updated this. It seems to have picked up some new links and deleted some orphaned ones.

Reply:
Highly recommend viewing this PR with `&w=1` to see actual TOC changes.

Reply:
Do you know what generates the TOC? I'm making similar changes

Reply:
Looks like it is a VSCode extension I just installed for Markdown called "Markdown All in One" (a feature I may be disabling, at least for `updateOnSave`)